.github/workflows/sec-scan.yml

```diff
@@ -0,0 +1,199 @@
+##
```

Comment:
I agree it would be good to scan container images, but I'm not a fan of the amount of copy-paste here and how much maintenance burden it will cause.

We either need to refactor container image building into reusable action first or even better, scan the actual latest container image we have already published, because even if fresh container image is not vulnerable, it says nothing about vulnerability of the latest published container image.

Reply:
This wasn't copy-paste from the official docs, i was just doing this to test the tools. The code will be refactored to only use the scan jobs, not the whole container building process. Either include in our current workflows or use reusable workflows.